### ISSUE TITLE
Consistent separators for crop names

### ISSUE THREAD

**Issue by abigailsnyder** [JGCRI/gcamdata#532]
In version 2, we would like to drop the use of underscores for the crop names. Underscores should be reserved for use in other delimiters.

This will impact Root_Tuber, biomass_grass, and biomass_tree. 

Additionally, biomass_grass and biomass_tree each have an AgSupplySector entry of biomass.  Is this intentional?

**Comment by kvcalvin** [comment]
I fully support this for version 2. Inconsistency is very problematic when processing data.

**Comment by kvcalvin** [comment]
For biomass_grass and biomass_tree, the `AgSupplySector` is intentionally the same...this is because both land types produce products sold in the biomass market.

**Comment by bpbond** [comment]
Will this be able to be done simply via one massive find-and-replace through both code and (possibly compressed) data files?

**Comment by kvcalvin** [comment]
Maybe? Probably? We'd definitely need to change a set of csv files that include crop mappings and sector name/information. Hopefully, that is it, but there maybe a few places in the R code that hard code crop names.

**Comment by abigailsnyder** [comment]
I introduced a constant for this in my most recent PR (https://github.com/JGCRI/gcamdata/pull/702) for some uniting and separating I did. It might come in handy whenever we (eventually...) get back around to changing the crop names for Root_Tuber, biomass_grass, biomass_tree. 
Consistent capitalization would be nice (MiscCrop, RootTuber, BiomassGrass, BiomassTree), but honestly non-repeated separators would be good enough.
